services/certificate-api/Dockerfile

```diff
@@ -1,9 +1,5 @@
 FROM node:lts-alpine
-RUN apk update && apk add --no-cache nmap && \
-    echo @edge http://nl.alpinelinux.org/alpine/edge/community >> /etc/apk/repositories && \
-    echo @edge http://nl.alpinelinux.org/alpine/edge/main >> /etc/apk/repositories && \
-    apk update && \
-    apk add --no-cache \
+RUN apk add \
```

Comment:
WHy did you change this?

Reply:
it was causing build failure on runner

Reply:
Will there be any issues because you have removed this.

Reply:
there shouldn't as we have e2e tests which are working fine